dist/models/APIManager.js

```diff
@@ -0,0 +1,18 @@
+export class APIManager{
+  getAllExpensesByGroup(from = '',to = ''){
+    let d1 = (from.length > 1) ? `&d1=${from}` : ''
+    let d2 = (to.length > 1) ? `&d2=${to}` : ''
+    return $.get(`/expenses/?grouping=true${d1}${d2}`)
+  }
+  getAllGroups(){
+    return $.get('/categories')
+  }
+  addExpence(data){
+    return $.post('/new', data)
```

Comment:
instead of "new", you should call it "expense".... just the fact that it is a post request makes it "new"...